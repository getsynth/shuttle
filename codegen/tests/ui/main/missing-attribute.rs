#[shuttle_codegen::main]
async fn missing_attriute(pool: PgPool, cache: Redis) -> ShuttleRocket {}
