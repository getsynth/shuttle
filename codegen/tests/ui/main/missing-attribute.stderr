error: resource needs an attribute configuration

         = help: Try adding a config like `#[shuttle_shared_db::Postgres]`

 --> tests/ui/main/missing-attribute.rs:2:27
  |
2 | async fn missing_attriute(pool: PgPool, cache: Redis) -> ShuttleRocket {}
  |                           ^^^^

error: resource needs an attribute configuration

         = help: Try adding a config like `#[shuttle_shared_db::Postgres]`

 --> tests/ui/main/missing-attribute.rs:2:41
  |
2 | async fn missing_attriute(pool: PgPool, cache: Redis) -> ShuttleRocket {}
  |                                         ^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/main/missing-attribute.rs:2:74
  |
2 | async fn missing_attriute(pool: PgPool, cache: Redis) -> ShuttleRocket {}
  |                                                                          ^ consider adding a `main` function to `$DIR/tests/ui/main/missing-attribute.rs`
